support-frontend/knip.config.js

```diff
@@ -13,6 +13,8 @@ module.exports = {
 	ignoreDependencies: [
 		// used in package.json
 		'@guardian/browserslist-config',
+		// knip has trouble telling that this is used
+		'@guardian/support-service-lambdas',
```

Comment:
Was this previously happening? Is there a reason for this?

Reply:
It wasn't happening previously because we used to have some imports in the form
@guardian/support-service-lambdas/modules/product-catalog/src/productCatalog

I've shortened these to the alias
@modules/product-catalog/productCatalog

And knip doesn't understand the alias. We could put it back to the longer import I guess and remove this?

Reply:
Happy to go whatever way yoo think. iIf was me think I would put back but does appear to be a Knip limitation unfortunately